operatorconfig/moduleconfig/authorization/v1.10.0/deployment.yaml

```diff
@@ -17,12 +57,22 @@ spec:
         csm: <NAME>
         app: proxy-server
     spec:
+      serviceAccount: proxy-server
       containers:
       - name: proxy-server
         image: <AUTHORIZATION_PROXY_SERVER_IMAGE>
         imagePullPolicy: Always
+        env:
+          - name: SENTINELS
+            value: "sentinel-0.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-1.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-2.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-3.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-4.sentinel.<NAMESPACE>.svc.cluster.local:5000"
```

Comment:
Can we try computing the sentinel node name dynamic? This assumes we have 5, but it would be nice to make this dynamic based on the replica count.

Reply:
Made the changes to read the sentinel value dynamically.